### PR TITLE
new icon: excel (plain)

### PR DESCRIPTION
**Pull request by Thomas-Boi**
Taken from #116. Work was done by @Pmz64.

### CODE REVIEW

**Review by amacado — CHANGES_REQUESTED** [review]
I'm not sure if Microsoft Excel should be part of this iconset. I unterstand that you can script within excel (macros/vb) but I'm not sure if it fit's within our guidelines. I would like to open a discussion about it.

**Comment by Thomas-Boi** [comment]
I was a bit iffy on this too but apparently, many businesses like small accounting or insurance firms still use Excel as a database of sort. Furthermore, I've heard learning VBA is a huge advantage => VBA runs in Excel so it makes Excel like an "editor" for VBA.

**Comment by Panquesito7** [comment]
Well, many developers use Excel, but its purpose is not mainly for developers. I'm not really sure what to do, but I guess we can remove it. There's no harm if we keep it or remove it. 🙂

File: icons/excel/excel-plain.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#1D7044" d="M128,108.2c0,0,0-84.7,0-90s-5.2-5-5.2-5H75.7V0L0,13.2v101.7L75.7,128v-14.6c0,0,41.9,0,47.2,0C128.1,113.4,128,108.2,128,108.2z M42.9,87.2l-7.3-17.9l-6.9,16.9L19,85.8l11.3-22.1L20.1,41.6l9.3-0.5l6.4,17l7.3-17.9l9.8-0.5L41.3,63.5l12,24.3L42.9,87.2z M123.6,109H76v-8.7h11.2V90.3H76v-6h11.2v-10H76v-5.8h11.2V58.1H76v-5.8h11.2V42.2H76v-5.7h11.2V26.4H76v-9.1h47.6L123.6,109L123.6,109z"/><rect x="93.1" y="26.4" fill="#1D7044" width="20.3" height="10.1"/><rect x="93.1" y="58.1" fill="#1D7044" width="20.3" height="10.4"/><rect x="93.1" y="42.2" fill="#1D7044" width="20.3" height="10.1"/><rect x="93.1" y="90.3" fill="#1D7044" width="20.3" height="10.1"/><rect x="93.1" y="74.3" fill="#1D7044" width="20.3" height="10"/></svg>
```
Comment (Panquesito7) [inline icons/excel/excel-plain.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#1D7044" d="M128,108.2c0,0,0-84.7,0-90s-5.2-5-5.2-5H75.7V0L0,13.2v101.7L75.7,128v-14.6c0,0,41.9,0,47.2,0C128.1,113.4,128,108.2,128,108.2z M42.9,87.2l-7.3-17.9l-6.9,16.9L19,85.8l11.3-22.1L20.1,41.6l9.3-0.5l6.4,17l7.3-17.9l9.8-0.5L41.3,63.5l12,24.3L42.9,87.2z M123.6,109H76v-8.7h11.2V90.3H76v-6h11.2v-10H76v-5.8h11.2V58.1H76v-5.8h11.2V42.2H76v-5.7h11.2V26.4H76v-9.1h47.6L123.6,109L123.6,109z"/><rect x="93.1" y="26.4" fill="#1D7044" width="20.3" height="10.1"/><rect x="93.1" y="58.1" fill="#1D7044" width="20.3" height="10.4"/><rect x="93.1" y="42.2" fill="#1D7044" width="20.3" height="10.1"/><rect x="93.1" y="90.3" fill="#1D7044" width="20.3" height="10.1"/><rect x="93.1" y="74.3" fill="#1D7044" width="20.3" height="10"/></svg>

```

**Review by Panquesito7 — APPROVED** [review]
The icon itself looks good, though. 👍

**Comment by Thomas-Boi** [comment]
Let's open this up in `Discussion` and see what people think

**Comment by cseelus** [comment]
Well, Excel is the reason many mere mortals have done reactive programming, probably without even realizing it: https://branch-blog.qlik.com/what-is-reactive-programming-a1e82cf28575

**Comment by Thomas-Boi** [comment]
Since there's not a lot of strong support for Excel, I think we can close this PR for now. If we ever need to get the icon (see someone make a good case for it), we can reopen this.

**Comment by Thomas-Boi** [comment]
I'll keep a local copy of the branch

**Pull request closed without merging**